src/turndown_test.js

```diff
@@ -255,9 +255,10 @@ describe("turndown", () => {
 
   it("should properly create youtube shortcodes from placeholder divs", async () => {
     const testId = "F3N5EkMX_ks"
-    const inputHTML = `<div class="${YOUTUBE_SHORTCODE_PLACEHOLDER_CLASS}">${testId}</div>`
+    const location = "https://example.com"
+    const inputHTML = `<div class="${YOUTUBE_SHORTCODE_PLACEHOLDER_CLASS}">${testId}|${location}</div>`
     const markdown = await html2markdown(inputHTML)
-    assert.equal(markdown, `{{< youtube ${testId} >}}`)
+    assert.equal(markdown, `{{< youtube ${testId} ${location}>}}`)
```

Comment:
Can we add a space after `${location}` so the URL doesn't butt up directly against the closing brace?